src/SmartComponents/Advisor/Advisor.js

```diff
@@ -48,6 +50,15 @@ const Advisor = ({ recStats, recStatsStatus, advisorFetchStatsRecs, advisorFetch
         }
     }];
 
+    if (advisorIncidentsStatus === 'rejected') {
+        return (
+            <TemplateCard appName='Advisor' data-ouia-safe>
+                <TemplateCardHeader title='Advisor recommendations' />
+                <TemplateCardBody><FailState appName='Advisor'/></TemplateCardBody>
```

Comment:
Can we keep keep this on the original return? lil duplication could be eliminated by simply conditionally returning templatecardbody (if status is anything but failed return what exists)

Reply:
we could, right now a few cards are checking the loading state if the status isn't fulfilled, I just didn't rewrite it to do something different

Reply:
fair enough, ill followup with some work for advisor, lets get this in